dotcom/package.json

```
@@ -40,6 +40,6 @@
 		"svelte-preprocess": "5.0.3",
 		"typescript": "5.0.4",
 		"vite": "4.3.4",
-		"vite-imagetools": "^4.0.13"
+		"vite-imagetools": "^5.0.0"
```

can you try the latest version?
```suggestion
		"vite-imagetools": "^5.0.2"
```

---

Will need to update the lockfile, too. Could you share a link to the changelog?

---

Also, hi, nice to see you here! Just seen that there's a SvelteKit image component in the works. Happy to help test things in this repo!

---

yeah, I'll let you update the lockfile

it might be a little while still before we get the image component out as we're currently discussing some things based on Rich's feedback, but would love some testers for it in a couple weeks hopefully!